app/views/devise/registrations/new.html.erb

```diff
@@ -1,28 +1,31 @@
-<h2>Sign up</h2>
+<div class="container justify-content-center">
+  <div class="col col-6"><h2>Sign up</h2>
```

Comment:
Move the h2 tag to a new line. I think it will need it's own row and container but test it out and see how it looks.